### DISCUSSION TITLE
Public link to the device group directory

### DISCUSSION THREAD

**Discussion by Avessalom608** [Ylianst/MeshCentral#5904] [Q&A] [answered]
How to make a public directory for a group of devices?

A directory that all administrators could see and public links could be shared in.
![image](https://github.com/Ylianst/MeshCentral/assets/40124505/419e67c0-69fc-4e83-817d-21d2bef67c19)

**Comment by si458** [comment, marked as answer]
i dont believe you can (i could be wrong?)
its only your personal 'My Files' drive and the folder `Public` within it thats shared publicly
you would have to look into making a feature request and we can see IF this can be added
